docs/guides/12-Vue-integration.md

```diff
@@ -0,0 +1,197 @@
+This guide will lead you through the process of integrating Airship in [Vue](https://vuejs.org/).
+
+## Including Styles
+Airship styles need to be globally included within our Vue application, so that we can use styles everywhere instead of including them inside any component scope. We are going to import them in the application's entry point via npm.
+
+```
+npm i @carto/airship-style
+```
+
+To import the styles, we need to include `airship.css` in the file:
+```js
+import '@carto/airship-style/dist/airship.css';
+```
+
+## Integrating Web Components
+First, we need to tell Vue to ignore elements starting with `as-`, which are our Airship components. We don't want Vue to raise an error everytime an Airship component is included in a template.
+
+```js
+import Vue from 'vue';
+
+Vue.config.ignoredElements = [/as-\w+/];
+```
+
+Second, we need to import and define our Airship components. As we did before with Airship styles, we import them from npm, and then we need to execute `defineCustomElements` function.
+
+```js
+import { defineCustomElements } from '@carto/airship-components';
+defineCustomElements(window);
+```
+
```

Comment:
Add a note telling that these two steps are better done in the main entry point file.